linuxNew/Jenkinsfile

```diff
@@ -50,7 +51,7 @@ def buildAndTest() {
             break;
     }
     for (os in ["Debian", "RedHat", "Suse"]) {
-        sh("./gradlew clean packageJdk${os} checkJdk${os} --parallel -PPRODUCT=${PRODUCT} -PPRODUCT_VERSION=${VERSION}")
+        sh("./gradlew packageJdk${os} checkJdk${os} --parallel -PPRODUCT=${PRODUCT} -PPRODUCT_VERSION=${VERSION}")
```

Comment:
It's been updated. So we don't need the for loop any more. 

`./gradlew --parallel check -PPRODUCT=${PRODUCT} -PPRODUCT_VERSION=${VERSION} --stacktrace` should be good?

Reply:
In the most recent refactor I've broken the jobs down into sub-jobs that in parallel for each version/distro.

Reply:
A second thought, with the parallel for each version is it possible or easy to only generate one version of installers with jenkinsfile?